policy/modules/services/cockpit.if

```diff
@@ -0,0 +1,350 @@
+	files_watch_root_dirs($2)
+	files_watch_var_dirs($2)
+	files_watch_var_lib_dirs($2)
+
+	cockpit_use_ws_fds($2)
+	cockpit_rw_ws_stream_socket($2)
+
+	init_watch_runtime($2)
+	init_watch_utmp($2)
+
+	userdom_dontaudit_execute_user_tmpfs_files($2)
+')
+
+########################################
+## <summary>
+##	Transition to the cockpit session domain.
+## </summary>
+## <param name="domain">
+## <summary>
+##	Domain allowed to transition.
+## </summary>
+## </param>
+#
+interface(`cockpit_domtrans_session',`
+	gen_require(`
+		type cockpit_session_t, cockpit_session_exec_t;
+	')
+
+	corecmd_search_bin($1)
+	domtrans_pattern($1, cockpit_session_exec_t, cockpit_session_t)
+')
+
+########################################
+## <summary>
+##	Allow specified domain to get status of cockpit service
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_status',`
+	gen_require(`
+		type cockpit_unit_t;
+		class service status;
+	')
+
+	allow $1 cockpit_unit_t:service status;
+')
+
+########################################
+## <summary>
+##	Allow specified domain to enable cockpit units
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_enabledisable',`
+	gen_require(`
+		type cockpit_unit_t;
+		class service { enable disable };
+	')
+
+	allow $1 cockpit_unit_t:service { enable disable };
+')
+
+########################################
+## <summary>
+##	Allow specified domain to start cockpit units
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_startstop',`
+	gen_require(`
+		type cockpit_unit_t;
+		class service { start stop };
+	')
+
+	allow $1 cockpit_unit_t:service { start stop };
+')
+
+########################################
+## <summary>
+##	Create, read, write, and delete the cockpick runtime symlink files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_manage_runtime_lnk_files',`
+	gen_require(`
+		type cockpit_runtime_t;
+	')
+
+	files_search_runtime($1)
+	manage_lnk_files_pattern($1, cockpit_runtime_t, cockpit_runtime_t)
+')
+
+## <summary>
+##	Inherit and use cockpit session file descriptors.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_use_session_fds',`
+	gen_require(`
+		type cockpit_session_t;
+	')
+
+	allow $1 cockpit_session_t:fd use;
+')
+
+########################################
+## <summary>
+##	Read and write cockpit session unnamed pipes.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`cockpit_rw_session_pipes',`
+	gen_require(`
+		type cockpit_session_t;
+	')
+
+	allow $1 cockpit_session_t:fifo_file rw_fifo_file_perms;
+')
+
+## <summary>
```

Comment:
Incomplete comment block.

Reply:
It too me a stupid amount of time to realize it was missing the line of '#'s above 'summary'.